ui/dashboard/src/pages/signin/signin.tsx

```diff
@@ -33,15 +36,20 @@ const SignIn = () => {
         <h1 className="text-gray-900 typo-head-bold-huge">
           {`Sign in to Bucketeer`}
         </h1>
-        {/* <p className="text-accent-red-500 typo-para-medium">
-          No account found for this Google Account. Please check the email
-          entered or try a different Google Account.
-        </p> */}
+        {isGoogleAuthError && (
+          <p className="text-accent-red-500 typo-para-medium">
+            {` No account found for this Google Account. Please check the email
```

Comment:
Let's implement the translation assets.
For now, please use English for both English and Japanese translations.